mitodl/open-discussions
# Added channel invitation backend

#### TODO
 

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #1400 

#### What's this PR do?
Adds a `ChannelInvitation` backend API that allows moderators

#### How should this be manually tested?
- Go to '/api/v0/:channel_name/invites/` and POST an email address in the form.
- You should receive an email. 
- When you signup with that email you should be added as a contributor and subscriber

NOTE: we're not supporting one-step invite + signup as per the issue and if they enter a different email they won't get subscribed (this is expected)
